g3/generate/commit/prompts/creator.py

```diff
@@ -0,0 +1,106 @@
+import re
+from typing import Optional
+
+from g3.domain.message_tone import MessageTone
+from g3.generate.commit.prompts.examples.node import node_sample
+from g3.generate.commit.prompts.examples.python import python_sample
+from g3.generate.commit.prompts.examples.ruby import ruby_sample
+from g3.git.gitinfo import GitInfo
+from g3.main import config
+
+PY_PATTERN = re.compile(".*py")
+JS_PATTERN = re.compile(".*js")
```

Comment:
Nit: perhaps we should extend the `js` pattern to match `.ts` files as well, since it's so common in node apps.